mcnp/mcnp_funcs.cpp

```diff
@@ -498,37 +520,53 @@ void write_lcad_old(std::ofstream &lcadfile)
       get_real_prop( vol, cellid, "imp.e", imp_e );
     }
 
-    lcadfile << cellid << " ";
+    // If no importances specified, default to neutron mode
+    if ( ! imp_n_needed && ! imp_p_needed && ! imp_e_needed ) {
```

Comment:
makes more sense to move this up to just after you search for the tags and set `imp_*_needed` rather than inside loop for each cell